mappings/net/minecraft/client/render/entity/model/PlayerModel.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_210 net/minecraft/client/render/entity/model/PlayerModel
```

Comment:
`class_210` is also subclassed by other Biped models, such as `ZombieModel`, so rather than `PlayerModel` should it be something more generic, like `BipedModel`?

Reply:
EntityModel? Maybe MobModel?